### PR TITLE
Add support for LocalConnectTimeoutMs and LocalRequestTimeoutMs on service-defaults CRD

### PR DESCRIPTION
**Pull request by erdanzhang**
Changes proposed in this PR:

Add maxInboundConnections to service-defaults CRD
fix https://github.com/hashicorp/consul-k8s/issues/1641
Checklist:
* Tests added 

Reference of similar PR before adding entries to serviceDefault: 
https://github.com/hashicorp/consul-k8s/pull/1437

### CODE REVIEW

**Comment by david-yu** [comment]
Thank you @erdanzhang. Could you also sign the CLA as well? We will take a look.

File: charts/consul/templates/crd-servicedefaults.yaml [hunk 1]
```diff
@@ -8,8 +7,8 @@ metadata:
   creationTimestamp: null
   name: servicedefaults.consul.hashicorp.com
   labels:
-    app: {{ template "consul.name" . }}
-    chart: {{ template "consul.chart" . }}
+    app: consul
+    chart: consul-helm
```
Comment (thisisnotashwin) [inline charts/consul/templates/crd-servicedefaults.yaml:11]: The changes to this file should be autogenerated by running `make ctrl-generate ctrl-manifests`

**Review by thisisnotashwin — COMMENTED** [review]
Hey @erdanzhang !! Thank you so much for your contribution. Have a comment about how the CRD in the templates folder has been generated.

**Comment by david-yu** [comment]
Hi @erdanzhang you would either sign with the email associated with your GitHub account or amend the author email on the git commits associated with that PR to match the address from their cla submission. These are the only ways to get the CLA to pass from chatting with the CLA team.

File: charts/consul/templates/crd-servicedefaults.yaml (hunk 1 above)
Comment (erdanzhang) [reply charts/consul/templates/crd-servicedefaults.yaml]: @thisisnotashwin Thanks for the comments!! Auto-gen code is cool
After I run `make ctrl-generate ctrl-manifests`,  there are few other files got changed as well
```
	modified:   charts/consul/templates/crd-peeringacceptors.yaml
	modified:   charts/consul/templates/crd-peeringdialers.yaml
	modified:   charts/consul/templates/crd-servicedefaults.yaml
	modified:   control-plane/api/v1alpha1/zz_generated.deepcopy.go
	modified:   control-plane/config/crd/bases/consul.hashicorp.com_servicedefaults.yaml
```
Could you help double check if the changes are intended?

**Comment by erdanzhang** [comment]
@david-yu @thisisnotashwin  Seems the CLA check still failed after I amended the git author email to the one used to sign the agreement.
I end up teleporting this PR to a new PR:
https://github.com/hashicorp/consul-k8s/pull/1647
Sorry about the friction, let's move our discussion to the new PR.

**Pull request closed without merging**